e2e-tests/js_scripts/0005-proofPath_rpc.js

```diff
@@ -108,19 +108,19 @@ async function run(nodeName, networkInfo, _args) {
         return ReturnCode.ErrWrongAttestationTiming;
     }
 
-    // For each proof, get its Merkle path and evaluate the root
-    const attId = parseInt(interestingAttId.data['id']);
-    for (verifier of verifiers) {
-        verifier.path = await api.rpc.poe.proofPath(attId, verifier.statementHash);
-        console.log(`##### proofPath RPC returned (proof ${verifier.name}): ` + JSON.stringify(verifier.path));
-        let checked = await verifyProof(verifier.path, publishedRoot);
-        console.log(`Proof ${verifier.name} checked: ${checked}`);
-        failed |= !checked;
-    }
-
-    if (failed) {
-        return ReturnCode.ErrAttProofFailedVerification;
-    }
+    // // For each proof, get its Merkle path and evaluate the root
```

Comment:
Has this been commented out intentionally?

Reply:
my bad, I was testing changes and I commented this part but forgot to undo it. 
I'll revert it